Sources/FeaturevisorSDK/InstanceOptions.swift

```diff
@@ -0,0 +1,31 @@
+import FeaturevisorTypes
+import Foundation
+
+public struct InstanceOptions {
+    
+    private static let defaultBucketKeySeparator = "."
+    
+    /// An instance of `InstanceOptions` with sane defaults. This is a singleton instance.
+    public static let `default` = InstanceOptions()
```

Comment:
is this here only for the sake of supporting tests?

Reply:
No, no, I borrow the idea of using the singleton from Apple (for instance configuration for URLSeesion) or from other SDKs (like Contentful which also has some configuration object). I didn't want to maintain this "big" init constructor and write there all the default assignments (I guess that @maxgallo faced this when doing this). The clients will use it like this:
```
let options: InstanceOptions = .default
options.datafileUrl = "https://featurevisor.com/datafilecontent.json"
options.logger = MyAwecomeLogger()
...
```
What do you think?